packages/cli/scripts/test.sh

```diff
@@ -11,6 +11,9 @@ npx hardhat deploy --network local --clear --quiet --no-confirm --instance test
 # Run tests
 cd ../../..
 npx mocha
+test_exit_code=$?
 
 # Kill node
 pkill node
+
+exit "$test_exit_code"
```

Comment:
This fixes the error from the tests, which was causing for the Github Action no never fail, regardless if the tests failed.